recipe/meta.yaml

```diff
@@ -23,12 +23,14 @@ requirements:
     - pip
   run:
     - python >=3.8
-    - anaconda-cli-base
-    - anaconda-cloud-auth
+    - anaconda-cli-base >=0.2
```

Comment:
Pins LGTM. We will need the actual Sha from the package. Assuming waiting for the real or from the autotick bot.